libs/utils/website/src/lib/pages/profile/profile.tsx

```diff
@@ -47,48 +55,55 @@ function Subscriptions() {
     subscription => subscription.deactivation
   )
 
-  const locationOrigin = typeof window !== 'undefined' ? location.origin : ''
-  const thisLocation = typeof window !== 'undefined' ? location.href : ''
-
   return (
-    <SubscriptionsWrapper>
-      <SubscriptionListWrapper>
-        <H4 component={'h1'}>Aktive Abos</H4>
-
-        <SubscriptionListContainer
-          successURL={`${locationOrigin}/profile/subscription`}
-          failureURL={thisLocation}
-          filter={subscriptions => subscriptions.filter(subscription => !subscription.deactivation)}
-        />
-
-        {hasDeactivatedSubscriptions && (
-          <DeactivatedSubscriptions>
-            <Link href="/profile/subscription/deactivated">Gekündete Abos anzeigen</Link>
-          </DeactivatedSubscriptions>
-        )}
-      </SubscriptionListWrapper>
-
-      <SubscriptionListWrapper>
-        <H4 component={'h1'}>Offene Rechnungen</H4>
-
-        <InvoiceListContainer
-          successURL={`${locationOrigin}/profile/subscription`}
-          failureURL={thisLocation}
-          filter={invoices =>
-            invoices.filter(
-              invoice => invoice.subscription && !invoice.canceledAt && !invoice.paidAt
-            )
-          }
-        />
-      </SubscriptionListWrapper>
-    </SubscriptionsWrapper>
+    <>
+      <SubscriptionsWrapper>
+        <SubscriptionListWrapper>
+          <H4 component={'h1'}>Aktive Abos</H4>
+
+          <SubscriptionListContainer
+            filter={subscriptions =>
+              subscriptions.filter(subscription => !subscription.deactivation)
+            }
+            failureURL=""
+            successURL=""
+          />
+
+          {hasDeactivatedSubscriptions && (
+            <DeactivatedSubscriptions>
+              <Link href="/profile/subscription/deactivated">Gekündete Abos anzeigen</Link>
+            </DeactivatedSubscriptions>
+          )}
+        </SubscriptionListWrapper>
+
+        <SubscriptionListWrapper>
+          <H4 component={'h1'}>Offene Rechnungen</H4>
+
+          <InvoiceListContainer
+            filter={invoices =>
+              invoices.filter(
+                invoice => invoice.subscription && !invoice.canceledAt && !invoice.paidAt
+              )
+            }
+            failureURL=""
+            successURL=""
+          />
+        </SubscriptionListWrapper>
+      </SubscriptionsWrapper>
+
+      <ProfileWrapper>
+        <H4 component={'h1'}>Profil</H4>
+
+        <PersonalDataFormContainer mediaEmail="abo@gruppetto-magazin.ch" />
```

Comment:
looks to me as if every media would get the gruppetto info mail now.

Reply:
Good catch